package.json

```diff
@@ -16,68 +16,69 @@
     "link-check": "repo-link-check -c config/link-check/config.yml",
     "link-check-diff": "repo-link-check -c config/link-check/config.yml -d",
     "link-check-dev-server": "repo-link-check -c config/link-check/config.yml -r http://localhost:3000",
-    "link-check-exclude": "repo-link-check -c config/link-check/config.yml --unusedPatternsOnly"
+    "link-check-exclude": "repo-link-check -c config/link-check/config.yml --unusedPatternsOnly",
+    "prepare": "husky install"
```

Comment:
result of following husky's install instructions again on each PR that had it upgraded, not sure if this is a v7->v8 thing or we just didn't have it set up right before.